editor/test/selectors.js

```diff
@@ -459,6 +462,54 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'getBlockSelectionStart', () => {
+		it( 'returns null if there is no multi selection', () => {
+			const state = {
+				editor: {
+					blockOrder: [ 123, 23 ],
+				},
+				multiSelectedBlocks: { start: null, end: null },
+			};
+
+			expect( getBlockSelectionStart( state ) ).to.be.null();
+		} );
+
+		it( 'returns multi selection start', () => {
+			const state = {
+				editor: {
+					blockOrder: [ 5, 4, 3, 2, 1 ],
+				},
+				multiSelectedBlocks: { start: 2, end: 4 },
```

Comment:
Yes, `start` has the index where the selection started, `end` where it ended. See also `getSelectedBlocks`.